core/state/intermediate_hashes.go

```diff
@@ -67,10 +63,7 @@ func (ih *IntermediateHashes) BranchNodeLoaded(prefixAsNibbles []byte, incarnati
 	trie.CompressNibbles(prefixAsNibbles, &buf.B)
 
 	var key []byte
-	if len(buf.B) > common.HashLength {
-		if incarnation == 0 {
-			panic("0 incarnation")
-		}
+	if len(buf.B) >= common.HashLength {
```

Comment:
Why do you change `>` to `>=` ?

Reply:
i have no good reason. removing this change.